hw3_calculator.py

```diff
@@ -0,0 +1,20 @@
+print('Введіть перше число ')
+number_1 = int(input())
```

Comment:
You can use
number_1 = int(input("Enter first number "))

Reply:
fixed